app/controllers/idv/phone_errors_controller.rb

```diff
@@ -45,5 +46,15 @@ def track_event(type:)
 
       analytics.idv_phone_error_visited(**attributes)
     end
+
+    def set_gpo_letter_available
+      def gpo_letter_available
+        @gpo_letter_available ||= FeatureManagement.enable_gpo_verification? &&
+                                  !Idv::GpoMail.new(current_user).mail_spammed? &&
+                                  !(sp_session[:ial2_strict] &&
+                                    !IdentityConfig.store.usps_upload_allowed_for_strict_ial2)
+
+      end
```

Comment:
Guessing the nested method isn't expected?

```suggestion
      @gpo_letter_available ||= FeatureManagement.enable_gpo_verification? &&
                                !Idv::GpoMail.new(current_user).mail_spammed? &&
                                !(sp_session[:ial2_strict] &&
                                !IdentityConfig.store.usps_upload_allowed_for_strict_ial2)
```

Reply:
nope, whoops